openshift/vault-manager.template.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: vault-manager
+objects:
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    labels:
+      app: vault-manager
+    name: vault-manager
+  spec:
+    replicas: ${{REPLICAS}}
+    strategy:
+      type: RollingUpdate
+      rollingUpdate:
+        maxUnavailable: 0
+        maxSurge: 1
+    selector:
+      matchLabels:
+        app: vault-manager
+    template:
+      metadata:
+        labels:
+          app: vault-manager
+      spec:
+        containers:
+        - image: ${IMAGE}:${IMAGE_TAG}
+          imagePullPolicy: Always
+          name: vault-manager
+          env:
+            - name: GRAPHQL_QUERY_FILE
+              value: ${GRAPHQL_QUERY_FILE}
+            - name: GRAPHQL_SERVER_URL
+              valueFrom:
+                secretKeyRef:
+                  key: gql.base_url
```

Comment:
Can the secret injector put two different secrets in the same environment context with a key?

```
gql.base_url
gql.username
gql.password
vault.auth_type
vault.role_id
vault.secret_id
```

If not, then this works:

```
```
base_url
username
password
auth_type
role_id
secret_id
```
```

Reply:
https://gitlab.cee.redhat.com/service/app-interface#manage-secrets-via-app-interface-openshiftnamespace-1yml-using-vault